pygenometracks/tests/test_maf.py

```diff
@@ -310,3 +311,30 @@ def test_second_maf_withe():
         os.remove(outfile.name)
     # Remove the index file:
     os.remove(os.path.join(ROOT, 'mm10_chr2_isl2_lessspe.maf.index'))
+
+
+def test_first_maf_seq_hg19_pdf():
+    extension = '.pdf'
+
+    outfile = NamedTemporaryFile(suffix=extension, prefix='pyGenomeTracks_test_',
+                                 delete=False)
+    ini_file = os.path.join(ROOT, "first_maf_seq_hg19.ini")
+    region = "9:128093930-128093970"
+    expected_file = os.path.join(ROOT, 'master_first_maf_seq_hg19.pdf')
+    # matplotlib compare on pdf will create a png next to it.
+    # To avoid issues related to write in test_data folder
+    # We copy the expected file into a temporary place
+    new_expected_file = NamedTemporaryFile(suffix='.pdf',
+                                           prefix='pyGenomeTracks_test_',
+                                           delete=False)
+    os.system(f'cp {expected_file} {new_expected_file.name}')
```

Comment:
shutil.copy(src, dst) could be used here.

Reply:
Right, thanks